s4e-web/src/app/views/map-view/map/map.component.ts

```diff
@@ -61,6 +62,21 @@ export class MapComponent implements OnInit, OnDestroy {
     zoomLevel: MapComponent.DEFAULT_ZOOM_LEVEL
   });
 
+  private _tileLoadFunction = (imageTile: ImageTile, src: string) => {
+    const xhr = new XMLHttpRequest();
+    xhr.open('GET', src);
+    xhr.responseType = 'arraybuffer';
+    // This ensures the authentication token is sent on CORS requests to {a-c}.maps domains.
+    // Such requests will nevertheless fail if the target domain doesn't expose Access-Control-Allow-Credentials=true header.
```

Comment:
I'd recommend to extract the comment outside function, so another developer cold find it easier.